src/views/YearsViews.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Only years
+ */
+
+import AtomicDateObject from '../models/AtomicDateObject';
+import DatePickerFactory from '../models/datePickerFactory/DatePickerFactory';
+import DatePickerView from './DatePickerViewInterface';
+import ViewConfiguration from '../enums/ViewConfiguration';
+import DatePickerBaseView from './DatePickerBaseView';
+
+export default class YearsView extends DatePickerBaseView implements DatePickerView {
+
+  constructor(model: DatePickerFactory, viewConfiguration: ViewConfiguration, grid: boolean) {
+    const continuousScroll = true;
+
+    // Until date ranges are implemented, looping is not possible with years.
+    // if (viewConfiguration.looping === true || viewConfiguration.continuousScroll === undefined) {
+    //   continuousScroll = false
+    // }
+    super(model, continuousScroll);
+
+    grid ? this.frameElementClassName = 'date-picker-grid' : this.frameElementClassName = 'date-picker-list';
```

Comment:
Less redundant: `this.frameElementClassName = grid ? 'date-picker-grid' : 'date-picker-list';`.

Reply:
excellent call!